components/director/cmd/director/main.go

```diff
@@ -468,3 +503,19 @@ func webhookService() webhook.WebhookService {
 
 	return webhook.NewService(webhookRepo, uidSvc)
 }
+
+func buildScheduler(namespace string, timeout time.Duration) (operation.Scheduler, error) {
+	cfg, err := rest.InClusterConfig()
+	if err != nil {
+		log.D().WithError(err).Warn("Could not start K8S Scheduler. Will use the default one")
+		return &operation.DefaultScheduler{}, nil
```

Comment:
Not sure we want to keep this default scheduler - it doesn't make sense to allow async operations with it. It was left out because we had to have a stub implementation before the actual one was implemented.

I think this should either throw an error here, or the directives responsible for determining what kind of operation is made (ASYNC or SYNC) it should throw an http client error if an ASYNC operation is requested but only the default scheduler is enabled.

Essentially my concern is that the default scheduler is not really any implementation at all.